app/views/users/show.html.erb

```diff
@@ -7,7 +7,7 @@
       <% card.body do %>
         <div class="flex items-center flex-col md:flex-row gap-4">
           <%= render User::AvatarComponent.new(current_user: current_user, classes: " w-24") %>
-          <h2 class="basis-3/5 text-2xl pl-4" data-test-id="profile_username"><%= current_user.username %></h2>
+          <h2 class="truncate w-full basis-3/5 text-2xl md:pl-4" data-test-id="profile_username"><%= current_user.username %></h2>
```

Comment:
It looks like w-full is making the name go all the way to the left on small screen sizes:

<img width="685" alt="Screenshot 2022-10-10 at 18 47 22" src="https://user-images.githubusercontent.com/7963776/194925289-17d82860-f8ee-495c-a774-227128bc4527.png">

But, easy enough sorted
```suggestion
          <h2 class="truncate w-full basis-3/5 text-2xl text-center md:text-left md:pl-4" data-test-id="profile_username"><%= current_user.username %></h2>
```